config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-postsubmits.yaml

```diff
@@ -24,4 +24,30 @@ periodics:
       # docker-in-docker needs privileged mode
       securityContext:
         privileged: true
+- name: ci-gcp-compute-persistent-disk-csi-driver-k8s-integration-stable-1-13-4
```

Comment:
do we want to just name it 1-13 for now? Then when we add support for choosing the latest stable, then we don't need to change the name again.

Reply:
I think 1-13-4 is fine for now. When we add support for choosing the latest stable we can rename it. I don't think renaming is that big of a deal (maybe it is, please correct me if I am wrong)